packages/client/src/pages/Game/Game.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC, useEffect, useRef } from 'react'
+import { Navbar } from '../../components/Navbar'
+import { requestAnimation, handleKeyDown } from './helpers'
+import { CANVAS_HEIGHT, CANVAS_WIDTH } from './constants'
+
+export const Game: FC = () => {
+  const canvasRef = useRef<HTMLCanvasElement>(null)
+
+  useEffect(() => {
+    const canvas = canvasRef.current
+    if (canvas === null) return
+    const canvasContext = canvas.getContext('2d')
+    if (canvasContext === null) return
```

Comment:
Рекомендую всё ж добавить хотя бы [отступы](https://learn.javascript.ru/coding-style#otstupy) (между логическими блоками):
```suggestion
    const canvas = canvasRef.current
    if (canvas === null) return

    const canvasContext = canvas.getContext('2d')
    if (canvasContext === null) return
```

Со скобками тоже рекомендуется чтобы лучше добавлять их, так как читается лучше, но и так приемлимо,
https://learn.javascript.ru/coding-style#figurnye-skobki

Reply:
Согласен, добавил отступ